packages/graphql-language-service-utils/package.json

```diff
@@ -21,6 +21,10 @@
   ],
   "main": "dist/index.js",
   "types": "src/index.ts",
+  "engines": {
+    "node": ">= 9.7.3"
```

Comment:
This could be a breaking change? We can drop Node 8 support from 1st Jan 2020, but might be too early right now?

Reply:
It will be a breaking change. Hmm... maybe we shouldn't use dynamic imports yet, and just stick with require until then? I wanted to deprecate it now, its so close!!

Reply:
If it's breaking anyway, might as well go whole hog?

Reply:
I'd just thought I'd leave it to `engineStrict: false` by default, so that it will warn in the case that its a browser build with node 8 which should still be supported.